### PR TITLE
Added pineapple-lemonade recipe

### PR DESCRIPTION
**Pull request by saikoushalg**

### CODE REVIEW

File: src/recipes/pineapple-lemonade.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        },
+        {
+            "quantity": "1",
+            "measure": "cup",
+            "ingredient": "water"
+        }
+        
+
+
+
```
Comment (alfg) [inline src/recipes/pineapple-lemonade.json:34]:
```suggestion
```

Can remove empty lines.

File: src/recipes/pineapple-lemonade.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        "Give a quick stir and serve chilled."
+       
+    ],
+    "image": "pineapple-lemonade.jpg",
+
+    "keywords": [
+        "fruit",
+        "sweet",
+        "refreshing",
+        "summer drink"
```
Comment (alfg) [inline src/recipes/pineapple-lemonade.json:50]:
```suggestion
        "summer"

```

**Review by alfg — CHANGES_REQUESTED** [review]
Hi @saikoushalg, just 2 suggestions. Thanks!

**Comment by saikoushalg** [comment]
hello! @alfg can you please check the PR once again thankyou.

**Comment by alfg** [comment]
Looks good! Thanks!

**Pull request merged**